config/features.yml

```diff
@@ -95,7 +95,7 @@ features:
     description: Enables disclaimer for possible untranslated content on spanish pages
     enable_in_development: true
   check_in_experience_lorota_security_updates_enabled:
```

Comment:
do we still need to keep it since we're not using it in the code anymore? is it used by website?

Reply:
Yeah, that's right.  It is used by the website for partial dial-up.